src/vs/workbench/contrib/positronIPyWidgets/browser/positronIPyWidgetsService.ts

```diff
@@ -354,8 +354,8 @@ class IPyWidgetsWebviewMessaging extends Disposable implements IIPyWidgetsWebvie
 	 *
 	 * @param message The message.
 	 */
-	postMessage(message: ToWebviewMessage) {
-		this._notebookRendererMessagingService.receiveMessage(
+	postMessage(message: ToWebviewMessage): Promise<boolean> {
```

Comment:
I didn't know that `postMessage` was async, and thought it might be related to the issue, but it wasn't. Probably worth keeping in.